frontend/src/components/Button/SimpleButton.tsx

```diff
@@ -3,8 +3,10 @@ interface SimpleButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement
 }
 
```

Comment:
Consider adding a short comment (or a new file in the doc folder), outlining the intended use of these buttons.